include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -110,7 +110,13 @@ namespace MGTransferGlobalCoarseningTools
 
   /**
    * Similar to the above function but also taking a @p policy for
-   * repartitioning the triangulations on the coarser levels.
+   * repartitioning the triangulations on the coarser levels. If
+   * @p preserve_fine_triangulation is set, the input triangulation is not
+   * altered,
+   * else the triangulation is coarsened. If @p repartition_fine_triangulation
+   * is set, the triangulation on the finest level is repartitioned as well. If
+   * both flags are set, the input triangulation is simply used as the finest
+   * triangulation.
```

Comment:
Isn't it that the first flag is `true` and the second `false`?

Reply:
Yes you are right! I have changed it! Thanks!